File: SideBarMenu/SideBarMenu/ViewController.swift
//
//  ViewController.swift
//  SideBarMenu
//
//  Created by CTS_DEP_AMM00217 on 30/11/16.
//  Copyright © 2016 CTS_DEP_AMM00217. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    @IBOutlet weak var openSideBar: UIBarButtonItem!
   
    @IBOutlet weak var mylabel: UILabel!
    var receivingData:String!
    var str:String!
    override func viewDidLoad() {
        super.viewDidLoad()
        str=receivingData;
        // Do any additional setup after loading the view, typically from a nib.
        openSideBar.target=self.revealViewController();
        openSideBar.action=Selector("revealToggle:");
        //add gesture recogniser
        //self.view.gestureRecognizers(self.revealViewController().panGestureRecognizer());
        self.view.addGestureRecognizer(self.revealViewController().panGestureRecognizer());
        mylabel.text=str;
        
       
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    


}

File: shareAppXamarin/shareAppXamarin/ViewController.cs
using System;

using UIKit;

namespace shareAppXamarin
{
	public partial class ViewController : UIViewController
	{
		protected ViewController(IntPtr handle) : base(handle)
		{
			// Note: this .ctor should not contain any initialization logic.
		}

		public override void ViewDidLoad()
		{
			base.ViewDidLoad();
			UILabel a = new UILabel
			{
				Text = CommonCode.add.addMethod(-1),
				Frame = new CoreGraphics.CGRect(50, 50, 200, 80)
			};
			View.AddSubview(a);

			// Perform any additional setup after loading the view, typically from a nib.
		}

		public override void DidReceiveMemoryWarning()
		{
			base.DidReceiveMemoryWarning();
			// Release any cached data, images, etc that aren't in use.
		}
	}
}
